src/js/action/notifyAction.js

```diff
@@ -0,0 +1,10 @@
+import Dispatcher from '../dispatcher/Dispatcher';
+
+const getNotification = (message) => {
+  Dispatcher.dispatch({
+    Type: 'GET-NOTIFY',
```

Comment:
GET-NOTIFICATION seems more appropriate. I suggest you make all your action types constants and refer to them.